gkr_iop/src/utils.rs

```diff
@@ -30,42 +27,42 @@ where
     E: ExtensionField,
 {
     let out_evals: Vec<_> = layer
-        .expr_evals
+        .out_eq_and_eval_exprs
         .iter()
         .flat_map(|(_, out_eval)| out_eval.iter())
         .collect();
+    // TODO: change to par_ter
```

Comment:
just a kindly reminder to change it back to `par_iter()` version :)

Reply:
Fixed in https://github.com/scroll-tech/ceno/pull/970/commits/5682bd995729ad270e9a4995fbf3f6880ad9ec0b. Thanks for catching that!

Reply:
Fixed in https://github.com/scroll-tech/ceno/pull/970/commits/5682bd995729ad270e9a4995fbf3f6880ad9ec0b. Thanks for catching that!